modules/clients/src/test/java/org/apache/ignite/internal/jdbc2/JdbcResultSetSelfTest.java

```diff
@@ -1128,4 +1131,65 @@ private TestObjectField(int a, String b) {
             return S.toString(TestObjectField.class, this);
         }
     }
+
```

Comment:
Let's fix the codestyle here and below. At the very least, we should remove the mention of non-existent classes from the javadoc.